src/test.ts

```diff
@@ -104,9 +109,39 @@ export function runTest(testDirectory: string, rulesDirectory?: string | string[
             };
         });
 
+        // test against fixed files
+        let fixedFileText: string;
+        let newFileText: string;
+        try {
+            const fixedFile = fileToLint.replace(/\.lint$/, FIXES_FILE_EXTENSION);
+            const stat = fs.statSync(fixedFile);
+            if (stat.isFile()) {
+                fixedFileText = fs.readFileSync(fixedFile, "utf8");
+                // accumulate replacements
+                const replacements = failures.reduce((acc, failure) => {
```

Comment:
I'm not sure the reduce here is worth the syntactic and cognitive cost, it might be simpler to just iterate?